nodejs/week2/src/backend/index.js

```diff
@@ -0,0 +1,102 @@
+const express = require("express");
+const app = express();
+const port = 3000;
+
+///meals///
+/*app.get("/meals", (request, response) => {
+  let mealsData = getDataFromJson(meals);
+  let reviewsData = getDataFromJson(reviews);
+  let mealsWithReviews = mealsData.map((meal) => {
+    meal.review = reviewsData.filter((review) => review.id === meal.id);
+    return meal;
+  });
+  response.send(mealsWithReviews);
+});*/
+
+const mealsRouter = require("./routes/meals.js");
+app.get("/meals", mealsRouter);
```

Comment:
The '/meals' path should be handled by a single router that should be able to handle all requests with different query params:

/meals => give me all meals
/meals?maxPrice=100 => give me all cheap meals under 100 crowns
/meals?title=chicken => give me all meals with chicken
/meals?title=chicken&maxPrice=100 => give me all cheap meals with chicken

(and all other params as detailed in the `homework.md` file)